metadata-ingestion/src/datahub/ingestion/source/sql/mssql/source.py

```diff
@@ -614,6 +617,23 @@ def construct_job_workunits(
         ).as_workunit()
         # TODO: Add SubType when it appear
 
+        path = [
+            BrowsePathEntryClass(id=data_job.entity.flow.env),
+            BrowsePathEntryClass(id=data_job.entity.flow.platform_instance or ""),
+            BrowsePathEntryClass(id=data_job.entity.flow.db),
+        ]
+
+        if data_job.entity.flow.schema:
+            path.append(BrowsePathEntryClass(id=data_job.entity.flow.schema))
+
+        path.append(BrowsePathEntryClass(id=data_job.entity.flow.name))
+
+        yield MetadataChangeProposalWrapper(
+            entityType=data_job.type,
```

Comment:
entityType can be inferred from entityUrn

```suggestion
```

Reply:
@hsheth2 This commit suggestion was implemented.